test/integration/conftest.py

```diff
@@ -82,27 +82,27 @@ def api_database(api_isolation: ApiTestEnvironment) -> ApiContextProvider:
     The test case optionally can pass a name and additional parameters for
     spawning the database:
 
-    def test_case(api_database):
-        with api_database(additional_parameters = ["--option"]):
+    def test_case(api_context):
+        with api_context(additional_parameters = ["--option"]):
             ...
     """
     return build_api_context_provider(api_isolation)
 
 
 @pytest.fixture(scope="module")
-def api_default_database_module(
+def api_default_env(
     api_isolation_module: ApiTestEnvironment,
```

Comment:
```suggestion
    api_isolation: ApiTestEnvironment,
```

Reply:
Not possible, because  `api_isolation_module` refers to the global fixture